packages/open-next/src/build.ts

```diff
@@ -377,6 +377,7 @@ function createStaticAssets() {
   // - .next/BUILD_ID => _next/BUILD_ID
   // - .next/static   => _next/static
   // - public/*       => *
+  // - src/app/favicon.ico => *
```

Comment:
A technicality 🤓 , it's a single file so the "*" is incorrect.

Reply:
yeah, I just didn't think it through. I'll fix it

Reply:
Thanks! Maybe not relevant now, but will we need to do this for `robots.txt`? We can kick that can until someone needs it.

Reply:
Next.js already bundles all the metadata files, and they are even accessible through SST deployments. So, I don't think there's a need to do the same for robots.txt. However, I did have some trouble getting opengraph-images to work in the same way. If you are open to adding this feature, I'm down to figure out a solution.